index.html

```diff
@@ -319,10 +318,10 @@ <h2>Contact me</h2>
                             I’d love to help with it!
                         </p>
 
-                        <form action="#" method="">
+                        <form action="https://formspree.io/html/" method="POST">
```

Comment:
- [ ] Good Job for Implementing the previous code reviewer suggestion of using [Formspree service](https://formspree.io/html) to collect the form data but you forget to use form `hashid({form_id})`  just like the following `form action="https://formspree.io/f/{form_id}" method="post"`
- When I submitted the form it's throws an below error :- 
![Capture](https://user-images.githubusercontent.com/93598189/229853569-91e12410-5692-4388-8e3c-6c9d7452dfec.PNG)